chainer/optimizer.py

```diff
@@ -384,8 +384,12 @@ def update(self, lossfun=None, *args, **kwds):
 
         """
         if lossfun is not None:
+            use_cleargrads = kwds.pop('use_cleargrads', False)
```

Comment:
I think it has no problem, but strictly speaking it breaks the compatibility. How do you think about adding a flag to the optimizer object itself?